### ISSUE TITLE
datepicker dialog example should not show disabled dates

### ISSUE THREAD

**Issue by jongund** [w3c/aria-practices#1121]
There are days in the first and last row of the calendar grid that are from the previous or next month. They are disabled for keyboard users. When clicked, they change the calendar to the previous or next month. Some users find this surprising behavior.

There has been discussion of having a click choose the date. But, those days are reported to screen reader users as disabled in reading mode, so making them clickable is not appropriate; they are disabled only for keyboard users.

The resolution the task force agreed upon is to remove those dates from the grid. All days in the grid will be in the month named in the dialog title. This is consistent with many date pickers found on production sites for hotels, airlines, etc.

**Comment by mcking65** [comment]
Resolved by commit e653a7a. Many thanks to @jongund!

**Comment by Rasubra8** [comment]
What would be the acceptable behaviour when there are disabled dates or unavailable dates like holidays in between? Can such dates receive arrow key focus? Especially If not the screen reader users may not be aware of their existence/status.

**Comment by JAWS-test** [comment]
@Rasubra8: See https://www.w3.org/TR/wai-aria-practices-1.2/#kbd_disabled_controls

I think disabled dates should be able to get focus and be marked with disabled or aria-disabled.
